test/mocha/mock.data.js

```diff
@@ -0,0 +1,39 @@
+const data = {};
+module.exports = data;
+
+const zcaps = data.zcaps = {};
+
+const zcap0 = {
+  '@context': 'https://w3id.org/security/v2',
+  id: 'urn:zcap:z19vWhR8EsNbWqvazp5bg6BTu',
+  controller: 'did:key:z6Mkkt1BWYLPAAXwYBwyVHAZkL94tgT8QbQv2SUxeW1U3DaG',
+  // eslint-disable-next-line max-len
```

Comment:
I think mockdata is a good use for these directives.

Reply:
That said,  I think we could just disable max-len at the top of this file to avoid all the noise.

Reply:
I am unsure of what you are asking me to change here.

Reply:
have a single `/* eslint-disable max-len */` at the top of the file instead of disabling it on all the lines.

Reply:
Done! Apologies, I didn't see your second comment initially.